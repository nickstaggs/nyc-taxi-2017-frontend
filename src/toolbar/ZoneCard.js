import React from 'react';
import './../App.css';

const styles = {

}

class ZoneCardView extends React.Component {
  renderZoneCard() {
    return (
      <div style={styles}>

      </div>
    );
  }
  render() {
    return this.renderZoneCard();
  }
}

class ZoneCardContainer extends React.Component {
  render() {
    return <ZoneCardView {...this.state} />;
  }
}

export default ZoneCardContainer;