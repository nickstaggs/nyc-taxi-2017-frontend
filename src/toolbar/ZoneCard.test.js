import React from 'react';
import { shallow } from 'enzyme';
import ZoneCard from './ZoneCard';

it('should render', () => {
    const view = shallow(<ZoneCard/>);

    expect(view).toMatchSnapshot();
});