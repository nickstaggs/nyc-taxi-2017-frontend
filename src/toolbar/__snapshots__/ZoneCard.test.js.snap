// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<ZoneCardView
  error={false}
  loading={false}
  message={null}
  submit={[Function]}
  validationError={false}
/>
`;
